cli/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "rack"
+version = "0.1"
+description = "Command-line interface to RACK"
+requires-python = ">=3.7"
+
+dependencies = [
+    "ase==3.22.1",
+    "attrs==23.1.0",
+    "certifi==2023.7.22",
+    "chardet==5.1.0",
+    "colorama==0.4.6",
+    "idna==3.4",
+    "importlib-metadata==6.7.0",
+    "jsonschema==4.17.3",
+    "pyrsistent==0.19.3",
+    "python-dateutil==2.8.2",
+    "PyYAML==6.0",
+    "requests==2.31.0",
+    "Pillow==9.4.0",
+    "plotly==5.15.0",
+    "semtk-python3 @ git+https://github.com/ge-semtk/semtk-python3@7995623e12e93b9b252200b9bd507949b8c0a430",
+    "six==1.16.0",
+    "tabulate==0.9.0",
+    "urllib3==2.0.3",
+    "zipp==3.15.0",
+]
+
+optional-dependencies.dev = [
+    "mypy==1.0.0",
+    "pylint==2.16.2",
+    "pytest-docker==1.0.1",
+    "pytest==7.2.1",
+    "typed-ast==1.5.4",
+    "types-PyYAML==6.0.12.6",
+    "types-requests==2.28.11.13",
+    "types-tabulate==0.9.0.0",
+]
```

Comment:
OK, I assume that `pip3 install .` and `pip3 install ".[dev]"` install these `dependencies` and `optional-dependencies.dev` respectively.

Reply:
You're quite right!